cli/src/main.rs

```diff
@@ -213,6 +216,23 @@ fn real_main() -> Result<(), Error> {
                 }
             }
 
+            if args.tweaks.enable_flips_and_rotates {
+                let mut transformed_examples: Vec<_> = vec![];
+                for example in &examples {
+                    let mut new_examples: Vec<_> = vec![
+                        example.clone().with_transformations(vec![Transformation::FlipH]).clone(),
+                        example.clone().with_transformations(vec![Transformation::Rot90]).clone(),
+                        example.clone().with_transformations(vec![Transformation::FlipH, Transformation::Rot90]).clone(),
+                        example.clone().with_transformations(vec![Transformation::Rot180]).clone(),
+                        example.clone().with_transformations(vec![Transformation::FlipH, Transformation::Rot180]).clone(),
+                        example.clone().with_transformations(vec![Transformation::Rot270]).clone(),
+                        example.clone().with_transformations(vec![Transformation::FlipH, Transformation::Rot270]).clone(),
+                    ];
+                    transformed_examples.append(&mut new_examples);
+                }
+                examples.append(&mut transformed_examples);
+            }
+
```

Comment:
I think this whole part can be made a lot simpler by just calling load_image for the example and doing the transforms here. That way, you don't actually need to change anything in the lib crate at all, as the CLI is the only part that needs to know that the example inputs need to be transformed, and ImageSource already supports just passing a raw image.

Reply:
Addressed in 2d0c83b1263fb57e693c7cb7942201e03caa2c8d